eventstream.go

```diff
@@ -457,6 +459,7 @@ type AuditLog struct {
 	ObjectID        string          `json:"objectId,omitempty" valid:"optional"`
 	ObjectType      string          `json:"objectType,omitempty" valid:"optional"`
 	ObjectNamespace string          `json:"objectNamespace" valid:"required~use publisher namespace if resource has no namespace"`
+	ObjectProperty  string          `json:"objectProperty,omitempty" valid:"optional"`
```

Comment:
What's the property for? Which one should be marked as properties? And what's the value?

Reply:
This is for account history, the property can be whatever that is updated/related to the object, for example, display_name,country,username, for query